zeq/publisher.h

```diff
@@ -60,16 +115,19 @@ class Publisher
      * constructor uri.
      *
      * @return the publisher URI.
+     * @todo change signature to return zeq::URI, needs downstream project
+     *       adaptions. Also make zeq::URI( const servus::URI& from ) explicit.
      */
     ZEQ_API const servus::URI& getURI() const;
 
-    /** @return the port to which this publisher is bound. */
-    ZEQ_API uint16_t getPort() const;
```

Comment:
Since this is a hard API change, people migrating code need to know what's the alternative. Document it in the Changelog or keep this function deprecated if possible.